labscript_utils/__init__.py

```diff
@@ -15,6 +15,7 @@
 import os
 import traceback
```

Comment:
Unused?

Reply:
Unused in this module, and I don't think any code is doing `from labscript_utils import traceback`